frontend/src/views/GMembers.vue

```diff
@@ -555,6 +556,17 @@ const visibleServiceAccountTableHeaders = computed(() => {
   return filter(serviceAccountTableHeaders.value, ['selected', true])
 })
 
+watch(namespace, () => {
```

Comment:
Not directly related to this PR but the `namespace` should be imported from the `authzStore`.